Tasks/NugetPublisher/task.json

```diff
@@ -33,8 +33,16 @@
             "name": "connectedServiceName",
             "type": "connectedService:Generic",
             "label": "NuGet Server Endpoint",
-            "required": true,
-            "helpMarkDown": "The NuGet server generic endpoint, set the key 'Password/Token Key' field to your NuGet API key"
```

Comment:
The inputs support radio buttons and visibility conditions. This will allow you to provide a cleaner UI experience rather. This way you can get rid of the qualifying "mutually exclusive" statements in the tooltips and the XOR logic in the script. You can grep the task.json files for "radio" to see examples.